src/sentry/api/endpoints/group_tagkey_details.py

```diff
@@ -32,7 +32,7 @@ def get(self, request, group, key):
 
         Returns details for given tag key related to an issue.
 
-        :pparam string issue_id: the ID of the issue to retrieve.
+        :pparam object group: the group to retrieve values for
```

Comment:
It's definitely confusing but I *think* we only refer to groups as issues in the API (this docstring turns into API docs AFAIK): https://docs.sentry.io/api/events/get-group-tag-key-details/

The URL uses groups for backward compat I guess? Anyway, not sure if it's a big deal or not but we may want to keep consistent.

Reply:
oops yeah, I see now. Will change it back